mmf/modules/detr/models/__init__.py

```diff
@@ -0,0 +1 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
I think these modules are specific for Unit model only and so these can be placed under `models/unit/..`. @apsdehal what do you suggest?

Reply:
Thanks! I'll move them to a folder under `mmf.models.unit_modules`.

Reply:
Thanks for the suggestion! I've moved them to `mmf.models.unit`